presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewPlanValidator.java

```diff
@@ -1,145 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can we please keep the same class name, is there any reason to change it?

Reply:
It becomes difficult to compare otherwise?

Reply:
Okay, I change the name back.

It's just the name is a bit confusing. The class doesn't seem to have much to do with validating plan.